src/aap_eda/core/validators.py

```diff
@@ -57,6 +59,49 @@ def check_if_de_exists(decision_environment_id: int) -> int:
 
 
 def check_if_de_valid(image_url: str, eda_credential_id: int):
+    parsed_url = urllib.parse.urlparse(image_url)
+    base_message = f"Image url {image_url} is malformed; "
```

Comment:
@jshimkus-rh This pattern of using gettext will help us down the road https://github.com/ansible/eda-server/blob/aaee25008fdd8f5f5bdf8e2c416039615cf004e4/src/aap_eda/core/validators.py#L137